neo/VM/InteropService.py

```diff
@@ -357,5 +355,11 @@ def stack_item_to_py(stack_item):
     elif isinstance(stack_item, ByteArray):
         return stack_item.GetBigInteger()
 
+    elif isinstance(stack_item, InteropInterface):
+        return stack_item.GetInterface()
+
+    elif isinstance(stack_item, Struct):
+        return stack_item.GetArray()
```

Comment:
Not sure, but should we convert the array items like for `Array` above?

    return [stack_item_to_py(item) for item in stack_item.GetArray()]

Reply:
I haven't done any testing with `Struct` yet, but that should work